integration/unplugin/src/index.ts

```diff
@@ -204,7 +204,7 @@ const civetUnplugin = createUnplugin((options: PluginOptions = {}) => {
       sourceMaps.set(path.resolve(process.cwd(), id), compiled.sourceMap);
 
       const jsonSourceMap = compiled.sourceMap.json(
-        path.basename(id.replace(/\.tsx$/, '')),
+        path.basename(id.replace(/\.[t]sx$/, '')),
```

Comment:
What is the purpose of this change? Is it supposed to be `[jt]`?

Reply:
This fixes the sourcemap `sources` key from being `main.civet.jsx` and makes it the correct `main.civet` when building using esbuild.

Reply:
> What is the purpose of this change? Is it supposed to be `[jt]sx`?

Oops, misread, yes it is supposed to be that. Now to figure out why it appeared to be working as is... :P